apps/webapp/src/components/staking/account/header/index.tsx

```diff
@@ -0,0 +1,81 @@
+import { getDisplayDenomFromView } from '@penumbra-zone/types';
+import {
+  Card,
+  CardContent,
+  TooltipProvider,
+  Tooltip,
+  TooltipTrigger,
+  TooltipContent,
+} from '@penumbra-zone/ui';
+import { AccountSwitcher } from '@penumbra-zone/ui/components/ui/account-switcher';
+import { ValueViewComponent } from '@penumbra-zone/ui/components/ui/tx/view/value';
+import { Stat } from './stat';
+import { ValueView } from '@buf/penumbra-zone_penumbra.bufbuild_es/penumbra/core/asset/v1/asset_pb';
+import { STAKING_TOKEN_METADATA } from '@penumbra-zone/constants';
+import { stakingSelector } from '../../../../state/staking';
+import { useStore } from '../../../../state';
+
+/**
+ * A default `ValueView` to render when we don't have any balance data for a
+ * particular token in the given account.
+ */
+const zeroBalanceUm = new ValueView({
+  valueView: {
+    case: 'knownAssetId',
+    value: {
+      amount: { hi: 0n, lo: 0n },
+      metadata: STAKING_TOKEN_METADATA,
+    },
+  },
+});
+
+/**
+ * The header of the account view, with an account switcher and balances of
+ * various token types related to staking.
+ */
+export const Header = () => {
```

Comment:
<img width="1273" alt="image" src="https://github.com/penumbra-zone/web/assets/1121544/22b2a3fe-03f2-43a1-80e7-6ee70d393c17">

Reply:
Not related to this PR, but I wonder why the account switcher appears to make a request every time it's toggled. I feel like we should only need to request all delegations once and just have the UI show filtered results of that.